frontend/src/components/DefenceBox/DefenceConfigurationInput.tsx

```diff
@@ -67,8 +66,8 @@ function DefenceConfigurationInput({
 				id={id}
 				content={value}
 				onContentChanged={setValue}
-				inputInvalid={inputInvalid}
-				validateInput={isInputInvalid}
+				valueInvalid={!isConfigValid}
+				validateInput={validateConfigValue}
```

Comment:
Sorry Doro 😓 

I've just realised that we don't need to pass this` validateInput` function prop into the Themed components at all, because we already trigger onContentChanged whenever the value changes. So we can handle validation in this component instead, and just pass the "isInvalid" prop into the Themed components.

```
function handleChange(value: string) {
  setValue(value);
  setIsConfigValid(validateNewInput(value));
}

...

<ThemedTextArea
  content={value}
  contentInvalid={!isConfigValid}
  onContentChanged={handleChange}
```

Reply:
yep, makes sense, I made the changes

Reply:
I'm going to make one annoying request... Sorry again!

As the other props use name "content" rather than "value", can we change the prop name for the two Themed components from "valueInvalid" to "contentInvalid", for consistency? Thanks 😊